sdk/monitor/Azure.Monitor.Query/src/LogsClient.cs

```diff
@@ -87,15 +89,80 @@ public virtual LogsBatchQuery CreateBatchQuery()
             return new LogsBatchQuery(_clientDiagnostics, _queryClient, _rowBinder);
         }
 
-        internal static QueryBody CreateQueryBody(string query, TimeSpan? timeSpan)
+        internal static QueryBody CreateQueryBody(string query, TimeSpan? timeSpan, LogsQueryOptions options, out string prefer)
         {
             var queryBody = new QueryBody(query);
             if (timeSpan != null)
             {
                 queryBody.Timespan = TypeFormatters.ToString(timeSpan.Value, "P");
             }
 
+            StringBuilder preferBuilder = null;
```

Comment:
For such a small string, allocating a `StringBuilder` is probably slower than interpolation or string concat.